app/views/application/robots.text.erb

```diff
@@ -0,0 +1,4 @@
+User-agent: *
+Allow: /
```

Comment:
I believe if we want to allow access to crawl all pages, the directive is to use Disallow and then leave it blank. The Allow directive is to override any defined Disallowed directives for the content we do want crawled. Otherwise, everything looks good.

Reply:
@kscali  Thanks, it should be fixed now!

Reply:
forgot to mention this robots file should have the sitemap Sitemap: https://tutoria.io/sitemap.xml @agiamona

Reply:
@leovcunha the robots file does reference the sitemap. Like the sitemap generator, it uses "#{request.protocol}#{request.host}/sitemap.xml" for the url to the sitemap instead of hardcoding https://tutoria.io/sitemap.xml. Let me know if there are any adjustments you’d like for me to make.